x-pack/metricbeat/module/aws/s3_daily_storage/_meta/fields.yml

```diff
@@ -0,0 +1,18 @@
+- name: s3_daily_storage
+  type: group
+  description: >
+    `s3_daily_storage` contains the daily storage metrics that were scraped from AWS CloudWatch which contains monitoring metrics sent by AWS S3.
+  release: beta
+  fields:
+    - name: bucket.name
+      type: keyword
+      description: >
+        Name of a S3 bucket.
+    - name: bucket.size.bytes
+      type: scaled_float
+      description: >
+        The amount of data in bytes stored in a bucket.
+    - name: number_of_objects
```

Comment:
Maybe `objects.total`? This way we follow a common structure with other fields in case we end up having things like `objects.avg.size.bytes` or anything like this.

Reply:
@sayden I'm hesitant to put `total` there because this value is actually an `average` statistic output of total number of objects from Cloudwatch. Also this metric in Cloudwatch is called `NumberOfObjects` so I was trying to make them as close as possible :-)

Reply:
Oh, I see. Thanks for the explanation :+1: :)